src/main/scala/dxWDL/compiler/IR.scala

```diff
@@ -22,8 +22,21 @@ object IR {
   val REORG = "reorg"
   val CUSTOM_REORG_CONFIG = "reorg_config"
 
+  // The following keywords/types correspond to app(let) attributes from dxapp.json (except for
+  // inputSpec/outputSpec attributes, which are defined separately). These attributes can be used
+  // in the meta section of task WDL, and will be parsed out and used when generating the native
+  // app.
+  val META_DESCRIPTION = "description"
+  val META_DEVELOPER_NOTES = "developer_notes"
+  val META_TYPE = "type"
```

Comment:
Can you make sure this doesn't conflict with with what DxNI generates? You can (and should) run the regression tests for this (staging, ./scripts/run_tests.py).